### PR TITLE
Spelling corrections

### PR DESCRIPTION
**Pull request by albert-github**
Spelling corrections `a e...` -> `an e...`

### CODE REVIEW

File: Algebraic_foundations/doc/Algebraic_foundations/Concepts/EuclideanRing.h [hunk 1]
```diff
@@ -21,7 +21,7 @@ Moreover, `CGAL::Algebraic_structure_traits< EuclideanRing >` is a model of
 
 \cgalHeading{Remarks}
 
-The most prominent example of a Euclidean ring are the integers.
+The most prominent example of an Euclidean ring are the integers.
```
Comment (afabri) [inline Algebraic_foundations/doc/Algebraic_foundations/Concepts/EuclideanRing.h:24]: Not sure about this one.  See [this discussion](https://forum.wordreference.com/threads/a-an-euclidean-eu.1067647/)

File: Algebraic_kernel_d/include/CGAL/Algebraic_kernel_d/Curve_pair_analysis_2.h [hunk 2]
```diff
@@ -613,7 +613,7 @@ class Curve_pair_analysis_2 :
     /*
      * \brief returns the indices of the <tt>i</tt>th event value
      *
-     * Returns a Event_indices <tt>(fg,ffy,ggy)</tt> such that
+     * Returns an Event_indices <tt>(fg,ffy,ggy)</tt> such that
```
Comment (afabri) [inline Algebraic_kernel_d/include/CGAL/Algebraic_kernel_d/Curve_pair_analysis_2.h:616]: Additionally Event_indices must be back ticked.

File: Arrangement_on_surface_2/include/CGAL/Arr_point_location/Arr_batched_point_location_traits_2.h [hunk 3]
```diff
@@ -411,7 +411,7 @@ class Arr_batched_point_location_traits_2 {
     }
   };
 
-  /*! Obtain a Equal_2 function object. */
+  /*! Obtain an Equal_2 function object. */
```
Comment (afabri) [inline Arrangement_on_surface_2/include/CGAL/Arr_point_location/Arr_batched_point_location_traits_2.h:414]: backtick

File: Arrangement_on_surface_2/include/CGAL/Arrangement_2/Arrangement_on_surface_2_global.h [hunk 4]
```diff
@@ -737,7 +737,7 @@ insert_non_intersecting_curve
 
   if (vh1 != nullptr) {
     if (vh2 != nullptr) {
-      // Both endpoints are associated with a existing vertices.
+      // Both endpoints are associated with an existing vertices.
```
Comment (afabri) [inline Arrangement_on_surface_2/include/CGAL/Arrangement_2/Arrangement_on_surface_2_global.h:740]: remove article as plural

File: Arrangement_on_surface_2/include/CGAL/Surface_sweep_2/Arr_basic_insertion_traits_2.h [hunk 5]
```diff
@@ -440,7 +440,7 @@ class Arr_basic_insertion_traits_2 {
     { return (m_base_eq(p1.base(), p2.base())); }
   };
 
-  /*! Obtain a Equal_2 function object */
+  /*! Obtain an Equal_2 function object */
```
Comment (afabri) [inline Arrangement_on_surface_2/include/CGAL/Surface_sweep_2/Arr_basic_insertion_traits_2.h:443]: backtick

File: Arrangement_on_surface_2/include/CGAL/Surface_sweep_2/Arr_overlay_traits_2.h [hunk 6]
```diff
@@ -705,7 +705,7 @@ class Arr_overlay_traits_2 {
     { return m_base_equal(xcv1.base(), xcv2.base()); }
   };
 
-  /*! Obtain a Equal_2 functor object. */
+  /*! Obtain an Equal_2 functor object. */
```
Comment (afabri) [inline Arrangement_on_surface_2/include/CGAL/Surface_sweep_2/Arr_overlay_traits_2.h:708]: backtick

File: Boolean_set_operations_2/include/CGAL/Boolean_set_operations_2/Gps_traits_decorator.h [hunk 7]
```diff
@@ -387,7 +387,7 @@ class Gps_traits_decorator
     }
   };
 
-  /*! Get a Equal_2 functor object. */
+  /*! Get an Equal_2 functor object. */
```
Comment (afabri) [inline Boolean_set_operations_2/include/CGAL/Boolean_set_operations_2/Gps_traits_decorator.h:390]: backtick

File: Number_types/include/CGAL/Sqrt_extension/Fraction_traits.h [hunk 8]
```diff
@@ -41,7 +41,7 @@ namespace Intern{
  *
  *  Extensions provide suitable specializations of \c CGAL::Fraction_traits.
  *  They are decomposable iff their coefficient type is.
- *  The denominator \e d of a Extension \e ext is a low common multiple
+ *  The denominator \e d of an Extension \e ext is a low common multiple
```
Comment (afabri) [inline Number_types/include/CGAL/Sqrt_extension/Fraction_traits.h:44]: The word Extension should be lower case.

File: Polynomial/include/CGAL/Polynomial_traits_d.h [hunk 9]
```diff
@@ -462,7 +462,7 @@ class Polynomial_traits_d_base< Polynomial< Coefficient_type_ >,
 
 
   // We use our own Strict Weak Ordering predicate in order to avoid
-  // problems when calling sort for a Exponents_coeff_pair where the
+  // problems when calling sort for an Exponents_coeff_pair where the
```
Comment (afabri) [inline Polynomial/include/CGAL/Polynomial_traits_d.h:465]: backtick

File: Algebraic_foundations/doc/Algebraic_foundations/Concepts/EuclideanRing.h (hunk 1 above)
Comment (albert-github) [reply Algebraic_foundations/doc/Algebraic_foundations/Concepts/EuclideanRing.h:24]: Did some checks as well looks indeed that in this case `a` is the right choice.

**Comment by albert-github** [comment]
@afabri Thanks for the extensive review, corrected g8iven (old) problems.

**Review by afabri — APPROVED** [review]

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0-Ic-39

**Pull request merged**